.idea/runConfigurations/templates/LabKey_Dev.xml

```diff
@@ -0,0 +1,10 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="LabKey Dev" type="Application" factoryName="Application">
+    <option name="MAIN_CLASS_NAME" value="org.apache.catalina.startup.Bootstrap" />
+    <module name="root.server.modules.platform.api" />
+    <option name="PROGRAM_PARAMETERS" value="start" />
+    <option name="VM_PARAMETERS" value="-Dcatalina.base=&quot;./&quot; -Dcatalina.home=&quot;./&quot; -Djava.io.tmpdir=&quot;./temp&quot; -Ddevmode=true -ea -Dsun.io.useCanonCaches=false -Xmx2G -classpath &quot;./bin/*PATH_SEPARATOR$APPLICATION_HOME_DIR$/lib/idea_rt.jar&quot;" />
+    <option name="WORKING_DIRECTORY" value="/usr/local/apache-tomcat-9.0.33" />
```

Comment:
Can/should this use CATALINA_HOME?

Reply:
Ah, good thought.  I'll try that.

Reply:
Unfortunately, this does not work. IntelliJ does not know anything about CATALINA_HOME (or other environment variables, apparently).  Strangely, if this value is defined to use `$CATALINA_HOME$` or, following the example from the file that is deleted `file://$CATALINA_HOME$`, it finds the oldest tomcat I have installed, so somewhere it knows how to look for something tomcat-like, but it's definitely not using my CATALINA_HOME environment variable.   Any ideas?

Reply:
Might be obvious, but IntelliJ has "Path Variables" (Settings -> Appearance & Behavior) that you can define within IntelliJ. I don't think they interact with environment variables. In the past, the run configurations have been defined to use these path variables. I believe IntelliJ will map these both directions (i.e., if you save LabKey_Dev it will reverse match the CATALINA_HOME path variable and replace the path with the variable name on save). I have CATALINA_HOME defined as a path variable, but it's mapped to an old value. I stopped bothering to update these values a while back.

Reply:
https://www.jetbrains.com/help/idea/2021.1/absolute-path-variables.html

Reply:
Thanks, @labkey-adam.  That's where I have this old tomcat version set.  We do still recommend this route for setting up your dev machine (https://www.labkey.org/Documentation/wiki-page.view?name=devMachine), so I'll update these files to use `$CATALINA_HOME$`.

Reply:
Agreed... definitely better than hard-coded. Probably works well for those who don't upgrade Tomcat once a month...